lib/fluent/env_utils.rb

```diff
@@ -0,0 +1,50 @@
+require 'fluent/config'
+
+module Fluent
+  module EnvUtils
+    def self.convert_env_var_to_type(value, type)
+      Fluent::Config.reformatted_value(type, value)
+    end
+
+    class OjOptions
+      OJ_OPTIONS = {
+        'bigdecimal_load': :symbol,
+        'max_nesting': :integer,
+        'mode': :symbol,
+        'use_to_json': :bool
+      }
+  
+      OJ_OPTIONS_ALLOWED_VALUES = {
+        'bigdecimal_load': %i[bigdecimal float auto],
+        'mode': %i[strict null concat json rails object custom]
+      }
+  
+      OJ_OPTIONS_DEFAULTS = {
+        'bigdecimal_load': :float,
+        'mode': :concat,
+        'use_to_json': true
```

Comment:
worth noting that this existing default is not actually doing anything according to the docs:

> :use_to_json [Boolean]

> Call to_json() methods on dump, default is false. The option is ignored in the :compat and :rails modes.

**The option is ignored in the :compat mode**